packages/react-components/src/lozenge/src/Lozenge.css

```diff
@@ -33,7 +33,16 @@
 }
 
 /* PRIMARY */
-.o-ui-lozenge-primary {
-    background-color: var(--primary-500);
-    color: var(--white);
+.o-ui-lozenge.o-ui-lozenge-primary {
+    background-color: var(--foreground-color-brand);
+    color: var(--static-white);
+}
+
+.o-ui-lozenge.o-ui-lozenge-primary .o-ui-lozenge-text {
+    color: var(--static-white);
+}
+
+.o-ui-lozenge.o-ui-lozenge-primary .o-ui-lozenge-icon {
+    fill: currentColor;
+    color: inherit;
```

Comment:
I don't fully understand your inherit strategy, I have a feeling it might not be fully consistant across all the components?

If I am following well in this case you should have provided `color: "inherit` to the Icon component instead of doing it in CSS

Reply:
Provide color: "inherit to a component - not anymore
o-ui-text-color-inherit - not anymore
we are lefe with these occasions where we need an icon to be the same color as the text while reacting to hover / focus / active states.